templates/create-user-job.yaml

```diff
@@ -38,19 +38,19 @@ spec:
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
           args:
-            - "airflow"
-            - "create_user"
-            - "-r"
+            - "users"
+            - "create"
+            - "--role"
```

Comment:
this commands will depend on the version (Master uses `users create` but 1.10.* uses `create_user`

Reply:
@kaxil unfortunately to merge this to master we'll need to make it 2.0 compliant, and THEN make it 1.10 complaint AGAIN when we cherry-pick to the 1-10-test branch.

Reply:
These args are fine on 1.10, it's just the new command names that are the issue.

I wonder if we should add the new commands to 1.10.11 -- it would make upgrading easier on people.